src/Tribe/Cache_Listener.php

```diff
@@ -32,7 +32,7 @@ public function init() {
 		 */
 		private function add_hooks() {
 			add_action( 'save_post', array( $this, 'save_post' ), 0, 2 );
-			add_action( 'updated_option', array( $this, 'update_last_save_post' ) );
+			add_action( 'updated_option', array( $this, 'update_last_save_post' ), 10, 3 );
```

Comment:
I don't think the reset of the param are used here and `10` is the default priority so it might be simpler to don't have the `10` and `3`.